### PR TITLE
upgrade to php7 compatibility as used in dragonbe/vies package

### PR DESCRIPTION
**Pull request by digibeuk**
version 1.0.7 en 1.0.8 of dragonbe/vies were only intended to be used with php7.0 this PR upgrades the code to be compatible with php7.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,14 +11,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "dragonbe/vies": "1.0.6",
+        "php": ">=7.0",
+        "dragonbe/vies": "^1.0",
+        "escapestudios/symfony2-coding-standard": "3.0",
```
Comment (Feijs) [inline composer.json]: Should packages  `escapestudios/symfony2-coding-standard`, `phpmd/phpmd`, `roave/security-advisories` and `squizlabs/php_codesniffer` not be `dev` requirements?

File: src/Validator/Constraint/VatNumber.php [hunk 2]
```diff
@@ -19,6 +19,14 @@
      */
     public $message = 'This is not a valid %format% vat number.';
 
+    /**
+     * @return string
+     */
+    public function getFormat()
```
Comment (Feijs) [inline src/Validator/Constraint/VatNumber.php]: You may now use strict return types :tada:

File: tests/Validator/Constraint/VatNumberValidatorTest.php [hunk 3]
```diff
@@ -61,30 +63,50 @@ public function testValidateWithEmptyValueWillReturnWithNoViolation()
         $this->api->expects(self::never())->method('validateVat');
         $this->context->expects(self::never())->method('addViolation');
 
-        self::assertNull($this->validator->validate(null, $this->constraint));
+        $this->validator->validate(null, $this->constraint);
     }
 
-    public function testValidateWithInValidVatNumberWillReturnWithViolation()
+    public function testValidateWithInValidConstraintWillReturnWithViolation()
```
Comment (Feijs) [inline tests/Validator/Constraint/VatNumberValidatorTest.php]: This test method name seems misleading, there is asserted *no* validation is added

File: composer.json (hunk 1 above)
Comment (digibeuk) [reply composer.json]: yes true!

**Review by Feijs — APPROVED** [review]

**Pull request merged**